pre_commit/clientlib/validate_manifest.py

```diff
@@ -25,6 +25,13 @@ class InvalidManifestError(ValueError):
             'language_version': {'type': 'string', 'default': 'default'},
             'files': {'type': 'string'},
             'expected_return_value': {'type': 'number', 'default': 0},
+            'stages': {
+                'type': 'array',
+                'default': [],
```

Comment:
Well, the way the logic is written, if it is an empty array, it will always be run in every stage. To me, this makes it more future proof since if we make another stage (say rebase), then it would still work on every stage by default. Of course, it is far more expressive to default it like you have suggested. I really don't mind changing it (this is not my project after all). Is your take on this still the same after my explanation?